### DISCUSSION TITLE
【社区奖励】龙年WTF贡献者红包 🧧 $30,000

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#626] [社区奖励]
WTF Academy自2022年4月1日建立已近2年，感谢成员们的陪伴和付出。截至2023年末，我们有一定资金盈余，见[链接](https://github.com/AmazingAng/WTF-Solidity/discussions/615)。现在正值春节时期，我们决定向社区贡献者发放 $30,000 红包（以USDT形式），分为三个主题：

1. 阳光贡献红包（$10,000）：平分给向WTF Academy的GitHub repo提过pr并被merge的贡献者，当然也包括贡献figma/视频/社区答疑的贡献者，共246人。

2. 杰出贡献红包（$10,000）：平分给为WTF Academy做出杰出贡献的贡献者，主要参考GitHub repo/Figma/视频/社区答疑记录，共32人。

3. 平台贡献红包（$10,000）：平分给持续建设[WTF Academy教育平台](https://wtf.academy)的贡献者，共9人。

下面是每项红包的名单公示（公示期7天），我们在统计时可能会有疏漏，如果漏掉了你的贡献，请谅解并留言/私信我。

领取方法：在2月28日前登录[WTF Academy官网](https://wtf.academy)，将钱包与GitHub账号绑定好，我们会在3月初发放红包。以Scroll链USDC的形式发出。

贡献者名单（GitHub和钱包）: https://docs.google.com/spreadsheets/d/1pCaLTkp8wpJeODiDRMya5NFvtRoUTSLVprtI15Vxo9k/edit?usp=sharing

### 阳光贡献名单（共245人）：

flyq, alex0207s, hattiepwvi, China-Chris, Mosamorphing, vecpeng, Doctor-DC, reborn-sama, AmazingAng, gofenix, 0xDjangoMo, chongqiangchen, zzyspace, overtrue, Glacier-Luo, 0xb00lean, DestinyWei, swittyle, cxp-13, jiandandkl, 22Xy, superboyiii, scguaquetam, 0xJackLee, Whisker17, MuuJian, yjshi2015, isol4te, push-edp, yanboishere, yutingzhao1991, MuHongWeiWei, ls-sudo, libaice, Hendry5479, zhangdiz, zaunww, hubingliang, seasidejuvenile666, holazz, Sylvia0907, geyu210, xiaofeiyang233, shalomxl, zhengqian8958, OpenWJY, Manuel-yang, 0xkawakarpo, cbgz121, lesenelir, lewislau86, cheng521521, Caffeine19, shengxinjing, Simon-He95, hcljsq, 0xKaso, ShawnQiang1, murrayee, KieSun, JustinAsdz, pseudoyu, beihaili, sugarforever, kangvcar, zh1phy, linc-code, Jarquer76, XX-Q, zhiyuan2007, CountryCousin, orangeryang, enehizy, hasselalcala, robertkodra, bowbowzai, shramee, Darlington02, 0x0077, 7feilee, kongtaoxing, greefea, Ha2ryZhang, eddieran, haddis3, shunjizhan, thurendous, AzumWatson, xiaoxiaocainiao, LeoZhaoZiao, huwangtao123, chiguayeshao, clark-cui, cong1223, jasonandjay, JacksonRGB, coolberwin, Confucian-e, 0x0918, simplezxh1, Skytrc, merdan-9, dev-ryan-ren, lukema95, ocandocrypto, pcdotfan, omahs, stoneworld, tankxu, 0xads, Pikkuherkko, Hderek22, JangoCCC, 0xNezha, songliling, digiwand, Lokiscripter, lcy101u, chrisguox, FFFFourwood, tangminjie, wishucry, XiaoYao-0, Rulesbreaker, jie1789, Hongchenglong, Eanam, HuiTaiLa, xxycfhb, RoyHonChain, SolidityGo, FlyingShuriken, buttonwild, kkthmyh, kosmosr, EasyChris, anbang, 0xc25fee20, magiconch, LIPUU, yechanggui, sdfgsdfsdf, wyn77, auzncn, quantum-proof, ISheepp, Ifdevil, ShuxunoO, 0x3c, x-player-001, y4000, chad779, Icarus9913, run-digitalx, LIYANG-UST, XdpCs, lilyang1989, u-u-z, shifenhutu, gasolin, 0xkookoo, itLeeyw, AeroXi, auroraug, PigCharid, alphafitz11, Cat2Boy, hotsjf, w7849516230, CryptoRbtree, Lee-TC, shao-shuai, isaacyuno, 0x794E33, Jiangyj19, 0xtangping, ben46, captpickle, C91F37, mookim-eth, luchenwei9266, BoZhao1992, SunWeb3Sec, finn79426, hitcxy, lzxeth, zhouxianyuan, jeasonstudio, shimmeris, wankhede04, PinelliaC, lujetliu, Silence-dream, Azleal, iavl, muzzby, yeecai, samaritanz, qiweiii, LaoBan-ywcm, LarryJBLee, charliex2, cjh20000613, SweetLoverFT, lxhyl, shadowDragons, DNCBA, eltociear, joeyzone, 0x3F97, Been101, Yumingyuan, w3-9, Ben0024, epitaphial, 0x4E33, yuhengfdada, deerhodl, AdamZeng1, yuler, beyond009, web3jt, riskers, w-zhong, WenYongqi, amesy, Hholz, time0x, skyMetaverse, wenyangliu, songlim327, Bot80926, keienWang, pkqs90, skywalker2017, LBJyang, lprowler, EthanOK, 0xAWM, MuHongWeiWei, 0xbitfly, LvisWang, Erin1314, KM-LEI, sdfgsdfsdf, matterturbulent, 0xhay

### 杰出贡献名单（共32人）：

Big-Aaron, flyq, Mosamorphing, reborn-sama, AmazingAng, 0xDjangoMo, chongqiangchen, Glacier-Luo, 0xb00lean, cxp-13, 22Xy, hattiepwvi, hubingliang, seasidejuvenile666, JustinAsdz, sugarforever, robertkodra, 0x0077, kongtaoxing, pcdotfan, EasyChris, buttonwild, 0xkookoo, SunWeb3Sec, XdpCs, 0xAWM , push-edp, magiconch, tankxu, stoneworld, 0xKaso, sdfgsdfsdf.

### 平台贡献名单（共9人）：

reborn-sama, AmazingAng, 0xDjangoMo, chongqiangchen, Glacier-Luo, 0xb00lean, tankxu, murrayee, stoneworld.

**Comment by Glacier-Luo** [comment]
祝WTF Academy越来越好！

**Comment by pcdotfan** [comment]
2024 做大做强！

**Comment by kongtaoxing** [comment]
祝 WTF Academy 龙年做大做强！

**Comment by XdpCs** [comment]
祝 WTF Academy 龙年 欣欣向龍

**Comment by 22Xy** [comment]
祝 WTF Academy 龙年一飞冲天！

**Comment by 0xDjangoMo** [comment]
WTF TO THE MOON!

**Comment by jasonandjay** [comment]
Continue BUILDING TO THE MOON!

**Comment by beyond009** [comment]
To the moon

**Comment by LvisWang** [comment]
WTF to da moon

**Comment by babbaa** [comment]
👍

**Comment by PigCharid** [comment]
祝WTF Academy越来越好！

**Comment by China-Chris** [comment]
祝wtf 龙年 如龙腾飞！！！

**Comment by lcy101u** [comment]
Fly me to the moon

**Comment by hotsjf** [comment]
谢谢wtf的空投！

**Comment by greefea** [comment]
刚才绑定钱包，还来得及领红包吗？
